index.js

```diff
@@ -115,6 +115,7 @@ function ParseServer(args) {
   router.merge(require('./installations'));
   router.merge(require('./functions'));
   router.merge(require('./schemas'));
+  router.merge(require('./global_config'));
```

Comment:
@drew-gross do you still want to me add `PARSE_EXPERIMENTAL_CONFIG_ENABLED` flag handling even though we might not need the `PUT`/`DELETE`?